lib/api-helper/project/fileCopy.ts

```diff
@@ -42,6 +42,10 @@ export interface FileMapping {
     donorPath: string;
     recipientPath: string;
 }
+export interface FileGlobMapping {
+    donorPath: string[];
```

Comment:
Yes, that was an option I looked into. But the `streamFiles` method takes an array of globs, some may be negations as well. So IMO it does not make sense to have a separate `recipientPath` per `FileGlobMapping`, as the array of globs given to streamFiles is one operation. 

The other option would be an array of `FileGlobMapping`, still having an array of donorPath inside, in order to keep the symmetry with the other methods (i.e. one path, one destination). But that just looked too complex.

Maybe better to rename `donorPath` to `globPatterns` in `FileGlobMapping` to avoid confusion.

Reply:
Makes sense.